### ISSUE TITLE
Move this repository to the Plone organisation?

### ISSUE THREAD

**Issue by fredvd** [collective/cookiecutter-plone-starter#104]
This template seems to have or is becoming the offical tool to create a new Plone 6 Setup.  Shouldn't we move it to the plone organisation on github?  I'm not sure if keeping this in the collective is beneficial to contributions vs. stability. 

It could also make people who follow the docs wonder why and what this collective is?

As a comparison, https://github.com/plone/cookiecutter-zope-instance  is already under plone (and as essential as this template, because called). 

cc: @ericof @sneridagh @mauritsvanrees @davisagli @gforcada @tkimnguyen

**Comment by davisagli** [comment]
+1 from me

**Comment by tkimnguyen** [comment]
I'm not aware of a reason not to, unless we wanted to be able to accept contributions from people who have not signed the contributor agreement. Who originally created this repo here? Perhaps they have some thoughts.

**Comment by stevepiercy** [comment]
> It could also make people who follow the docs wonder why and what this collective is?

It did for me. Yes, it's easier to create repos in collective, but it is also not super clear what to do once you get to this repo.

If you are intuitive, you might go to the GitHub organization page: https://github.com/collective/

There you might see a tiny link in the description to go to https://collective.github.io/ where you can learn what is this collective thing.

You will not get this information in the repo's readme: https://github.com/collective/cookiecutter-plone-starter

The only potential issue I see is that this repo is under the MIT license https://github.com/collective/cookiecutter-plone-starter/blob/main/LICENSE versus Plone's GPL https://plone.org/foundation/copyright-licensing-logo/license-faq. I know diddly squat about licensing issues when transferring ownership.

**Comment by davisagli** [comment]
> The only potential issue I see is that this repo is under the MIT license versus Plone's GPL

GPL is the default choice for repositories owned by the Plone Foundation, but there is precedent for some repositories being MIT licensed: https://github.com/orgs/plone/repositories?q=license%3AMIT&type=all&language=&sort= -- IIRC that requires approval by the board.

**Comment by sneridagh** [comment]
+1 for me.

**Comment by ericof** [comment]
I agree with the move, and I will plan it over the next few days (as we need to make sure the move is well documented and has deprecation warnings for people that still point here)

**Comment by ericof** [comment]
I just created the new repository on [plone/cookiecutter-plone](https://github.com/plone/cookiecutter-plone), which will consolidate all templates.
